src/Model/Build/BitbucketBuild.php

```diff
@@ -295,40 +295,4 @@ public function reportError(
 
         parent::reportError($builder, $plugin, $message, $severity, $file, $lineStart, $lineEnd);
     }
-
```

Comment:
Why? Method `getDiffLineNumber` used in method `reportError` (line `273`).

Reply:
Oops removed that function from the wrong file when I was cleaning up the code for the initial push, will fix that.

Reply:
When cleaning up code for initial commit, (for functionality not implemented in this first patch) I removed getDiffLineNumber from the wrong file it appears.